docs/install/pip.md

```diff
@@ -91,9 +92,37 @@ Once installed, check that everything is as it should be:
 
 ### Run GrimoireLab
 
-TBD
+In order to run GrimoireLab, you need to have Elasticsearch, Kibiter and MySQL running in your machine. For the
+sake of simplicity, we assume that they run in localhost with no authentication needed.
```

Comment:
How about adding this to the prerequisites section above?

Reply:
Done! feel free to merge if the new version is OK for you.